src/js/edu-benefits/5490/config/form.js

```diff
@@ -537,6 +537,31 @@ const formConfig = {
             educationProgram: {
               name: {
                 'ui:title': 'Name of school, university, or training facility you want to attend'
+              },
+              educationType: {
+                'ui:options': {
+                  updateSchema: (pageData, form, schema) => {
```

Comment:
@annekainicUSDS An example of `updateSchema` can be found in the schemaform docs [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/docs/schemaform/form-config.md#i-want-to-change-the-options-of-a-dropdown-based-on-some-other-field-data) (I've a PR out changing the variable names to be more descriptive, though).

@raquelromano-gov I'm all for using this PR instead! I wasn't too comfortable changing the reducer without @jbalboni's review, but I'm pretty sure it'll be alright; there are only two places that use `updateSchema` right now, and they're both here in the 5490, and the change will only slow down the `GET_DATA` action _(insert proper term here)_ by < 3 ms (on my machine, it was usually < 1 ms).

That said, I'm more than happy to hold off on it for now in case I missed something. Taking the the keys from the UI object is a clever workaround :smile: